wizard_builder/view_helpers.py

```diff
@@ -60,6 +60,8 @@ def _parse_questions(self, form):
     def _parse_answers(self, question, answer):
         if question.get('type') == 'Singlelinetext':
             self._append_text_answer(answer, question)
+        if question.get('type') == 'Textarea':
+            self._append_text_answer(answer, question)
         else:
             self._append_list_answers(answer, question)
```

Comment:
the successive `if` statements here are a bug risk, a safer way to do this would be

```python
if question.get('type') in ['Singlelinetext', ''Textarea']:
    self._append_text_answer(answer, question)
```